app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from contextlib import contextmanager
+
+
+@contextmanager
+def open_file(name: str, method: str) -> None:
+    func = open(name, method)
+    try:
+        yield func
+    finally:
+        func.close()
```

Comment:
You can do without a separate `open_file` function by using the built-in context manager `open` for this